PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Core/Models/WorkingHours.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using PrivateHospitals.Core.Models.Users;
+
+namespace PrivateHospitals.Core.Models;
+
+public class WorkingHours
+{
+    [Key] public int HoursId { get; set; }
+    
+    [ForeignKey("Doctor")]
+    public string DoctorId { get; set; }
+    public Doctor Doctor { get; set; }
+    
+    public DateOnly Date { get; set; }
+    public TimeSpan StartTime { get; set; }
+    public TimeSpan EndTime { get; set; }
```

Comment:
I don't understand what DateOnly means here. Do we store all working hours of the doctor during all the future years?
I suggest:
1. Don't create a separate table for it, but store it inside doctor as JSONB property
2. Store working hours per days like Monday, 8:00 to 17:00, etc.

This way we avoid creating another table and joins. On the other hand we will require handling doctor holidays in the future, for example, but let's not care about it in the scope of our tasks